### ISSUE TITLE
EGPSR command TakeI

### ISSUE THREAD

**Issue by JulioCCruzE** [kyordhel/GPSRCmdGen2015#27]
Hello

I have trouble with this command: 

$takeI    = $vbtake the {aobject? meta: Place to } from the {placement}

Because is an incomplete command our robot try to get information about of what object grasp. But I don't understand the part:  "meta: Place to"

Could you explain me what is supposed to do with this sentence?, Thank you.

**Comment by johaq** [comment]
I believe aobject will be replaced by an actual object from the known set. The meta thing is a hint to the referees to place the object at the generated location.

**Comment by kyordhel** [comment]
Indeed, meta is to give hints to the referees. Look inside the Common folder, documentation is there.
